app/livechat/server/lib/Livechat.js

```diff
@@ -627,6 +627,17 @@ export const Livechat = {
 		const visitor = LivechatVisitors.findOneById(room.v._id);
 		const agent = Users.findOneById(room.servedBy && room.servedBy._id);
 
+		const externalCustomFields = () => {
+			try {
+				const customFields = JSON.parse(settings.get('Accounts_CustomFields'));
+				return Object.keys(customFields)
+					.filter((customFieldKey) => customFields[customFieldKey].sendToIntegrations === true)
+					.map((field) => field);
```

Comment:
I'm sorry, I didn't notice this before, but that `.map` isn't doing anything right? 

```suggestion
					.filter((customFieldKey) => customFields[customFieldKey].sendToIntegrations === true);
```

Reply:
@sampaiodiego you're right, there was no reason for that `.map` method, my bad.
I just removed it, thanks.